js/timeout_error.go

```diff
@@ -66,3 +63,16 @@ func (t TimeoutError) Hint() string {
 	}
 	return hint
 }
+
+// ExitCode returns the coresponding exit code value to the place.
+func (t timeoutError) ExitCode() errext.ExitCode {
+	// TODO: add handleSummary()
```

Comment:
Why leave this for later?

Reply:
because I didn't want to add any new exit codes in this PR, and to be honest, I'm not sure this deserves its own separate exit code, `genericTimeout` is probably good enough :man_shrugging: We only have 128 exit codes, we should be somewhat conservative with them